app/components/HeaderBabyMenu.tsx

```diff
@@ -182,7 +182,7 @@ const HeaderBabyMenu = (props: any): any => {
               <ProfileSectionView>
                 <Heading3>{data.childName}
                   {genderName != '' && genderName != null && genderName != undefined ? <Heading5 style={styles.heading5Fontwg}>{', ' + genderName}</Heading5> : 
-                  <Heading5 style={styles.heading5Fontwg}>{', ' + 'Girl'}</Heading5>
+                  <Heading5 style={styles.heading5Fontwg}>{', ' + t('chilGender2')}</Heading5>

```

Comment:
Instead of naming chilGender1 or chilGender2 we should name as 
genderMale
genderFemale

Reply:
This is string variable name we used in string file. If we change we have to change in all language file.